voi/user/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
```

Comment:
Wrong, you should inherited from Using a custom user model when starting a project[¶](https://docs.djangoproject.com/en/4.2/topics/auth/customizing/#using-a-custom-user-model-when-starting-a-project)